bundles/net.tourbook/src/net/tourbook/weather/WeatherUtils.java

```diff
@@ -63,21 +62,19 @@ public static String buildFullWeatherDataString(final float temperatureValue,
                                                    final int humidityValue,
                                                    final float precipitationValue,
                                                    final float snowFallValue,
-                                                   final long time,
-                                                   final String timeZoneId) {
+                                                   final TourDateTime tourDateTime) {
 
-      final TourDateTime tourDateTime = TimeTools.createTourDateTime(time * 1000L, timeZoneId);
       final String tourTime = String.format("%3s", tourDateTime.tourZonedDateTime.getHour() + UI.UNIT_LABEL_TIME); //$NON-NLS-1$
 
-      final String temperature = String.format("%5s", Math.round(UI.convertSpeed_FromMetric(temperatureValue) * 10.0) / 10.0) //$NON-NLS-1$
+      final String temperature = String.format("%5s", roundDoubleToFloat(UI.convertTemperatureFromMetric(temperatureValue))) //$NON-NLS-1$
             + UI.UNIT_LABEL_TEMPERATURE;
```

Comment:
I think there are too many digits for the temperature/speed values

![image](https://user-images.githubusercontent.com/1283445/158318976-46b07b0a-9444-4e7b-95f2-1127c27aba20.png)


Can you format these values with `net.tourbook.common.formatter.FormatManager` for one tour, that the user can decide, how it is displayed. There is not yet a formatter for temperatures.

![image](https://user-images.githubusercontent.com/1283445/158319010-1f96a4d4-da0d-4957-8104-0f1757e71adb.png)

Reply:
> I think there are too many digits for the temperature/speed values
> 
Fixed

![image](https://user-images.githubusercontent.com/25235665/158440124-cd390dd7-c8fd-4a26-8726-03730fb82054.png)

> 
> Can you format these values with `net.tourbook.common.formatter.FormatManager` for one tour, that the user can decide, how it is displayed. There is not yet a formatter for temperatures.
> 

Good idea, done:
![image](https://user-images.githubusercontent.com/25235665/158439956-6ec2f186-e3f7-4f01-86bc-e9d562e0f31c.png)